raw-cms-app/src/modules/core/views/configuration-details-view/configuration-details-view.js

```diff
@@ -31,6 +32,8 @@ const _ConfigurationDetailsView = async (res, rej) => {
         this.title = isNew
```

Comment:
`title` could be a local variable here and its definition could be removed from `data`, like you did e.g. in file "raw-cms-app/src/modules/core/views/lambda-details-view/lambda-details-view.js"